ksqldb-engine/src/test/java/io/confluent/ksql/planner/plan/QueryProjectNodeTest.java

```diff
@@ -72,6 +72,24 @@
       .valueColumn(K, SqlTypes.STRING)
       .build();
 
+  private static final LogicalSchema INTERMEDIATE_SCHEMA = LogicalSchema.builder()
```

Comment:
I know this is super hacky, but I have like an hour left of my internship. @AlanConfluent WDYT about this workaround to the tests? It's related to your schema changes for SPQs and pull queries.